x-pack/plugins/fleet/server/plugin.ts

```diff
@@ -115,6 +117,7 @@ export interface FleetStartDeps {
   encryptedSavedObjects: EncryptedSavedObjectsPluginStart;
   security: SecurityPluginStart;
   telemetry?: TelemetryPluginStart;
+  savedObjectTaggingStart: SavedObjectTaggingStart;
```

Comment:
`savedObjectTaggingStart` is always undefined, any idea what am I missing?
It was added here https://github.com/elastic/kibana/pull/130774